.github/workflows/main.yml

```diff
@@ -33,31 +59,62 @@ jobs:
 
   build-n-test-n-coverage:
     name: Build, test and code coverage
+
+    needs: check_duplicate_runs
+    if: ${{ needs.check_duplicate_runs.outputs.should_skip != 'true' }}
+
     runs-on: ${{ matrix.os }}
+
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: [3.6, 3.7, 3.8, 3.9]
+
     env:
       OS: ${{ matrix.os }}
-      PYTHON: "3.9"
```

Comment:
I have no idea why this env variable was set because matrix build can setup up another python version.

Reply:
Probably carry over when we migrated from the other CIs.